src/components/datasets/DatasetOverview/DatasetOverview.vue

```diff
@@ -351,8 +351,9 @@
         ref="markdownEditor"
         :value="changelogText"
         :is-editing="isEditingMarkdown2"
-        :is-saving="isSavingMarkdown"
+        :is-saving="isSavingMarkdown2"
```

Comment:
@annaleeswitek is there no better naming option we can use like isSavingMardownNew? Or is this naming already part of a larger pattern?

Otherwise looks okay, won't let it hold you up.